app/code/community/Bolt/Boltpay/Helper/GeneralTrait.php

```diff
@@ -95,18 +95,22 @@ public function getItemImageUrl($item)
         /** @var Mage_Catalog_Helper_Image $imageHelper */
         $imageHelper = Mage::helper('catalog/image');
 
+        /** @var Mage_Catalog_Model_Product $product */
+        $product = Mage::getModel('catalog/product');
+
         /** @var Mage_Catalog_Model_Product $_product */
-        $_product = $item->getProduct();
+        $_product = $product->load($product->getIdBySku($item->getSku()));
```

Comment:
@adentong  
I have a couple of observations for M1 style.  Your intent is to use the object on line 99 strictly as utility, consider naming it `$productModel` to unambiguously define its intent and distinguish it from `$_product`.

Secondly, after the execution of line 102, `$_product` and `$product` reference the same object instance, so there is no need for a `$_product` variable.

In essence, line 102 could simply be 
```
$product->load($product->getIdBySku($item->getSku()));
```
`load` is returning $this for fluency.

Reply:
@mccottry I see. Will update.

Reply:
Done.